orcid-web/src/main/resources/freemarker/includes/ng2_templates/funding-details-ng2.ftl

```diff
@@ -144,6 +144,12 @@
                     <div class="content" *ngIf="fundingService.details[funding.putCode.value] != undefined">  
                         <span class="dotted-bar"></span>    
                         <div class="row">        
+                            <div class="org-ids" *ngIf="funding?.disambiguatedFundingSourceId?.value">
+                                <div class="col-md-12">   
+                                    <strong><@orcid.msg 'workspace_affiliations.organization_id'/></strong><br>
```

Comment:
@lizkrznarich I am using the same title "Organization identifiers" from the affiliations. Is this ok, or do we need to create a new translation for the fundings section?

Reply:
@leomendoza123 Same translation is fine!